tests/ipp-tests.test

```diff
@@ -168,7 +168,7 @@ DEFINE SUPPLY_REGEX "/^(type\=[A-Za-z]+|(maxcapacity\=([0-9]|\-){0,1})(level\=([
 #
 # Required by: PWG 5100.14 Section 5.x
 {
-	NAME "I-9. Get-Printer-Attributes Operation (default)"
```

Comment:
The (default) was in reference to not specifying the "requested-attributes" attribute... OK either way but this changes the name of the test which might require an errata of 5100.20...